cmd/deploy/cfghandler.go

```diff
@@ -113,6 +115,14 @@ func (ch *defaultConfigMapHandler) UpdateEnvsFromCommands(ctx context.Context, n
 	return nil
 }
 
+func (ch *defaultConfigMapHandler) AddPhase(ctx context.Context, name, namespace, phase string, duration time.Duration) error {
```

Comment:
For the future, out of scope of this PR. If we have 2 different implementations to satisfy an interface and some of the functions have duplicated the logic, it is error prone. We should extract those implementations to a common piece and the implementations should use that common piece through struct embedding or dependency